dev/NavigationView/Strings/af-ZA/Resources.resw

```diff
@@ -141,6 +141,10 @@
     <value>Terug-knoppie</value>
     <comment>Automation name for the nav view provided back button</comment>
   </data>
+  <data name="NavigationBackButtonToolTip" xml:space="preserve">
+    <value>Terug</value>
+    <comment>ToolTip caption for the back button</comment>
```

Comment:
why do we need there resw changes ?

Reply:
As mentioned above.  It was part of a merge.

Reply:
use "git checkout origin/master dev/NavigationView/Strings/af-ZA/Resources.resw" to revert the file to its original form. Do this for all such changes that were bad merges.